pkg/cli/subcommands/config/command.go

```diff
@@ -186,12 +190,28 @@ func getConfigValue(log log.Component, config config.Component, cliParams *cliPa
 		return err
 	}
 
-	value, err := c.Get(cliParams.args[0])
+	resp, err := c.GetWithSources(cliParams.args[0])
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("%s is set to: %v\n", cliParams.args[0], value)
+	fmt.Printf("%s is set to: %v\n", cliParams.args[0], resp["value"])
+
+	if cliParams.source {
+		sourcesVal, ok := resp["sources_value"].([]interface{})
+		if !ok {
+			return fmt.Errorf("failed to cast sources_value to []map[interface{}]interface{}")
+		}
+
+		fmt.Printf("sources and their value:\n")
+		for _, sourceVal := range sourcesVal {
+			sourceVal, ok := sourceVal.(map[string]interface{})
```

Comment:
Once `valueWithSource` is public we could use it here maybe ?

Reply:
Unfortunately I don't think so, the element can only be casted to `map[string]interface{}`
Perhaps because there is a HTTP call in between?